### ISSUE TITLE
fails auto commit when using husky

### ISSUE THREAD

**Issue by AndysonDK** [wearerequired/lint-action#293]
This is my lint-action configuration:

*.github/workflows/lint.yml*
```yml
name: Lint
on: [push, pull_request]

jobs:
  commitlint:
    name: Commit message linter
    runs-on: ubuntu-latest

    steps:
      - name: Check out repository code
        uses: actions/checkout@v2
        with:
          fetch-depth: 0
      - name: Lint commit messages
        uses: wagoid/commitlint-github-action@v4

  lintcode:
    name: Source code linter
    runs-on: ubuntu-latest

    steps:
      - name: Check out repository code
        uses: actions/checkout@v2
      - name: Set up node 14
        uses: actions/setup-node@v2
        with:
          node-version: 14
          cache: "yarn"
      - name: Install dependencies listed in package.json
        run: yarn install
      - name: Run linters
        uses: wearerequired/lint-action@v1
        with:
          auto_fix: true
          commit_message: "chore(style): fix code style issues with ${linter}"
          neutral_check_on_warning: true
          stylelint: true
          stylelint_extensions: css
          prettier: true
          prettier_extensions: json,md,yml
          eslint: true
          eslint_args: '--ignore-pattern "node_modules/.*" --max-warnings 0'
          eslint_extensions: js,ts,ts
```

And here is the husky file:
*.husky/prepare-commit-msg*
```bash
#!/bin/sh
. "$(dirname "$0")/_/husky.sh"

exec < /dev/tty && yarn cz --hook || true
```

When `lint-action` finds errors and fixes them it will auto commit them, but when it does that I get the following error

![image](https://user-images.githubusercontent.com/28491857/131157496-b6a67f08-dbff-4148-a963-d157eac82531.png)

Any ideas of how to fix this?

**Comment by ocean90** [comment]
The next version will have an option for skipping git hooks. For now you can use the workaround mentioned in #279.

**Comment by AndysonDK** [comment]
@ocean90 Awesome! When can I expect the new version to be released?

**Comment by AndysonDK** [comment]
> The next version will have an option for skipping git hooks. For now you can use the workaround mentioned in #279.

Doing what is stated in the PR as a workaround doesn't work since those changes (delete commit hooks) will also get committed which you don't want

**Comment by AndysonDK** [comment]
As a true workaround I'm just going to use the master branch directly as that contains the newest code
